products/fedora.md

```diff
@@ -32,18 +42,14 @@ releases:
 
 -   releaseCycle: "37"
     releaseDate: 2022-11-15
-    # releaseDate(37) + 13 months for now
-    # Revise once 39 comes out
     # Update with exact date found on https://docs.fedoraproject.org/en-US/releases/eol/ when EOL
-    eol: 2023-12-15
+    eol: 2023-12-05
```

Comment:
```suggestion
    eol: 2023-12-07
```

Why not this ?

Reply:
Because the description text below says "4 weeks", i.e. the 5th is Tuesday in 4 weeks. What you are suggesting is one month but not 4 weeks. I don't know which is right but if it is the month we should also update the description below.

Reply:
Here it also says four weeks, but I don't know which is actually correct: https://docs.fedoraproject.org/en-US/releases/lifecycle/#_maintenance_schedule

Reply:
Reviewing the last 4 reviews with official eol dates on https://docs.fedoraproject.org/en-US/releases/eol/ it is really 4 weeks, always released and eol'd on a Tuesday.

Reply:
Ok, thanks for the clarification. Will update the comment / description accordingly so there are no more questions about this in the future.